app/components/HelpText.ts

```diff
@@ -4,6 +4,8 @@ const HelpText = styled.p<{ small?: boolean }>`
   margin-top: 0;
   color: ${(props) => props.theme.textSecondary};
   font-size: ${(props) => (props.small ? "14px" : "inherit")};
+  word-break: break-all;
```

Comment:
Is word break required? I'm not sure we want a situation where words are split across lines:

![image](https://user-images.githubusercontent.com/380914/151735220-11917c60-a035-4c77-9ba9-6f309c376c09.png)